src/components/Calendar/RangePickerController.spec.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { RangePickerController } from '.';
+
+describe('RangePickerController', () => {
+  /**
+   * Style tests.
+   */
+  it('should render with default styles', () => {
+    const actual = shallow(<RangePickerController />);
+    expect(actual).toMatchSnapshot();
+  });
+
+  /**
+   * Accessibility tests.
+   */
+
+  /*
+  a11y test for RangePickerController failes (timeots most of the time)
+  for some reason. Should already be tested, anyway.
```

Comment:
I'm sure @connor-baer will tell you about this 😁

Reply:
Hm, I don't know what could be causing the timeout. 🤔

The `CalenderWrap` component has accessibility testing already, and so does the `DayPickerRangeController` I assume? If that's the case, I think it's okay to omit the accessibility test here.

Reply:
When I run the test separately it can pass, but it takes about a minute so there is no chance when part of the test. @connor-baer, yes the `DatePickerRangeController` is the `react-dates` export. I should probably remove the other two a11y tests just for speedup.